lib/typings/utils.ts

```diff
@@ -0,0 +1,34 @@
+import { MaterialUiPickersDate } from './date'
+import { isEqual, isAfter, endOfDay, setHours } from 'date-fns';
+
+export interface Utils {
+  date(value: any): MaterialUiPickersDate;
+  addDays(value: MaterialUiPickersDate, count: Number): MaterialUiPickersDate;
+  isValid(value: MaterialUiPickersDate): boolean;
+  isEqual(value: MaterialUiPickersDate, comparing: MaterialUiPickersDate): boolean;
+  isNull(value: MaterialUiPickersDate): boolean;
+  isAfter(value: MaterialUiPickersDate, comparing: MaterialUiPickersDate): boolean;
+  isAfterDay(value: MaterialUiPickersDate, comparing: MaterialUiPickersDate): boolean;
+  isBeforeDay(value: MaterialUiPickersDate, comparing: MaterialUiPickersDate): boolean;
+  isBeforeYear(value: MaterialUiPickersDate, comparing: MaterialUiPickersDate): boolean;
+  isAfterYear(value: MaterialUiPickersDate, comparing: MaterialUiPickersDate): boolean;
+  startOfDay(value: MaterialUiPickersDate): MaterialUiPickersDate;
+  endOfDay(value: MaterialUiPickersDate): MaterialUiPickersDate;
+  format(value: MaterialUiPickersDate): string;
```

Comment:
can you document what strings this function gets for formats (e.g. `'YYYY'`, `'MMM D'`, ...). helps others implement their own utils.

Reply:
@alitaheri I think we should leave pure functions for rendering dates. Like in persian-utils now. It would be more convenient to users, who would be override existsing utils :)

Reply:
format. Takes a second argument. The format string. I'm only saying it would be helpful to document the expected input fot format strings. How can that make it impure?

Reply:
Oh, I didnt understand you, sorry)